src/producer.rs

```diff
@@ -36,13 +36,14 @@ pub struct GCNOStem {
     pub llvm: bool,
 }
 
+#[cfg(not(windows))]
 fn clean_path(path: &PathBuf) -> String {
-    let path = path.to_str().unwrap().to_string();
-
-    #[cfg(windows)]
-    let path = path.replace("\\", "/");
+    path.to_str().unwrap().to_string()
+}
 
-    path
+#[cfg(windows)]
+fn clean_path(path: &PathBuf) -> String {
+    path.replace("\\", "/")
```

Comment:
Aargh, it looks like this has to be converted to a string first (see the failure on Appveyor)

Reply:
Ah, thanks! Just pushed a fix.